message/validation/consensus_validation.go

```diff
@@ -235,8 +232,7 @@ func (mv *messageValidator) validateQBFTMessageByDutyLogic(
 	if signedSSVMessage.SSVMessage.MsgID.GetRoleType() != spectypes.RoleCommittee { // Rule only for validator runners
 		for _, signer := range signedSSVMessage.GetOperatorIDs() {
 			signerStateBySlot := state.GetOrCreate(signer)
```

Comment:
also not protected

Reply:
You meant the next line with `signerState := signerStateBySlot.Get(msgSlot)`? I added a mutex there. 

If you meant this line (`signerStateBySlot := state.GetOrCreate(signer)`), could you please explain why you think it's not?

Reply:
you're right, `GetOrCreate` is indeed protected.